config.m4

```diff
@@ -1,4 +1,5 @@
 PHP_ARG_ENABLE(ddtrace, whether to enable Datadog tracing support,[  --enable-ddtrace   Enable Datadog training support])
+PHP_ARG_ENABLE(ddtrace-debug, whether ddtrace should be built in debug mode,[  --enable-ddtrace-debug Build Datadog tracing in debug mode],, no)
```

Comment:
Do you expect our clients to enable debug mode through pecl install ?

Reply:
I think they do but I don't have any data to back that up. :D The main thing is that it's quite common for extensions to provide a way to enable debug mode via a config flag (i.e., `--enable-apcu-debug` for APCu, `--enable-xdebug-dev` for Xdebug, etc) So it's just something that seems to be "the PHP way" to do it. :)